salty-747/html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/Boeing_FMC.js

```diff
@@ -26,7 +26,7 @@ class Boeing_FMC extends FMCMainDisplay {
     Init() {
         super.Init();
         this.maxCruiseFL = 450;
-        this.cruiseFlightLevel = 100;
+        this.cruiseFlightLevel = NaN;
```

Comment:
instead of assigning NaN, i'd just leave it as undefined by not initializing the variable (or alternatively assign it to null)